Workingdir: /home/rkohli/aml-project/PC-DARTS
Started at Mon Sep  9 15:19:49 CEST 2019
Running job darts+bohb-model using 2 cpus per node with given JID 3241994 on queue meta_gpu-black
Experiment dir : eval-EXP-20190909-152824
Worker running
09/09 03:29:19 PM DISPATCHER: started the 'discover_worker' thread
09/09 03:29:19 PM WORKER: start listening for jobs
09/09 03:29:19 PM DISPATCHER: started the 'job_runner' thread
09/09 03:29:19 PM DISPATCHER: Pyro daemon running on 127.0.0.1:43749
09/09 03:29:19 PM DISPATCHER: discovered new worker, hpbandster.run_01.worker.metagpu8.17282140225950078784
09/09 03:29:19 PM HBMASTER: adjusted queue size to (0, 1)
09/09 03:29:19 PM DISPATCHER: A new worker triggered discover_worker
09/09 03:29:19 PM HBMASTER: starting run at 1568035759.926127
09/09 03:29:20 PM WORKER: start processing job (0, 0, 0)
09/09 03:29:20 PM gpu device = cuda:0
09/09 03:29:20 PM config = {'drop_path_prob': 0.02061460579245411, 'grad_clip_value': 7, 'initial_lr': 0.00011663439741039343, 'n_conv_layers': 4, 'optimizer': 'adam', 'scheduler': 'Cosine', 'weight_decay': 0.00035880370373513694}
48 48 16
48 64 32
64 128 64
128 256 64
09/09 03:29:44 PM param size = 0.213546MB
09/09 03:29:44 PM WORKER: registered result for job (0, 0, 0) with dispatcher
09/09 03:29:44 PM job (0, 0, 0) failed with exception
Traceback (most recent call last):
  File "/home/rkohli/anaconda3/lib/python3.7/site-packages/hpbandster/core/worker.py", line 206, in start_computation
    result = {'result': self.compute(*args, config_id=id, **kwargs),
  File "darts-bohb.py", line 95, in compute
    if config['lr_schedular'] == 'Cosine':
KeyError: 'lr_schedular'

09/09 03:29:45 PM WORKER: start processing job (1, 0, 0)
09/09 03:29:45 PM gpu device = cuda:0
09/09 03:29:45 PM config = {'drop_path_prob': 0.08788897864685562, 'grad_clip_value': 6, 'initial_lr': 0.0003952703361071693, 'n_conv_layers': 5, 'optimizer': 'sgd', 'scheduler': 'Cosine', 'weight_decay': 0.0005546470606205093, 'nesterov': 'False', 'sgd_momentum': 0.011292463490771728}
48 48 16
48 64 32
64 128 32
128 128 64
128 256 64
09/09 03:29:45 PM param size = 0.245898MB
09/09 03:29:45 PM WORKER: registered result for job (1, 0, 0) with dispatcher
09/09 03:29:45 PM job (1, 0, 0) failed with exception
Traceback (most recent call last):
  File "/home/rkohli/anaconda3/lib/python3.7/site-packages/hpbandster/core/worker.py", line 206, in start_computation
    result = {'result': self.compute(*args, config_id=id, **kwargs),
  File "darts-bohb.py", line 95, in compute
    if config['lr_schedular'] == 'Cosine':
KeyError: 'lr_schedular'

09/09 03:29:45 PM WORKER: start processing job (2, 0, 0)
09/09 03:29:45 PM gpu device = cuda:0
09/09 03:29:45 PM config = {'drop_path_prob': 0.38609406437028876, 'grad_clip_value': 7, 'initial_lr': 0.0001907334943134668, 'n_conv_layers': 4, 'optimizer': 'sgd', 'scheduler': 'Exponential', 'weight_decay': 5.4672639600795444e-05, 'nesterov': 'False', 'sgd_momentum': 0.89688055400376}
48 48 16
48 64 32
64 128 64
128 256 64
09/09 03:29:45 PM param size = 0.213546MB
09/09 03:29:45 PM WORKER: registered result for job (2, 0, 0) with dispatcher
09/09 03:29:45 PM job (2, 0, 0) failed with exception
Traceback (most recent call last):
  File "/home/rkohli/anaconda3/lib/python3.7/site-packages/hpbandster/core/worker.py", line 206, in start_computation
    result = {'result': self.compute(*args, config_id=id, **kwargs),
  File "darts-bohb.py", line 95, in compute
    if config['lr_schedular'] == 'Cosine':
KeyError: 'lr_schedular'

09/09 03:29:45 PM WORKER: start processing job (3, 0, 0)
09/09 03:29:45 PM gpu device = cuda:0
09/09 03:29:45 PM config = {'drop_path_prob': 0.022565871002635164, 'grad_clip_value': 4, 'initial_lr': 3.911864231516776e-06, 'n_conv_layers': 5, 'optimizer': 'sgd', 'scheduler': 'Cosine', 'weight_decay': 4.4471211239117325e-05, 'nesterov': 'False', 'sgd_momentum': 0.11139393663484437}
48 48 16
48 64 32
64 128 32
128 128 64
128 256 64
09/09 03:29:45 PM param size = 0.245898MB
09/09 03:29:45 PM WORKER: registered result for job (3, 0, 0) with dispatcher
09/09 03:29:45 PM job (3, 0, 0) failed with exception
Traceback (most recent call last):
  File "/home/rkohli/anaconda3/lib/python3.7/site-packages/hpbandster/core/worker.py", line 206, in start_computation
    result = {'result': self.compute(*args, config_id=id, **kwargs),
  File "darts-bohb.py", line 95, in compute
    if config['lr_schedular'] == 'Cosine':
KeyError: 'lr_schedular'

09/09 03:29:45 PM WORKER: start processing job (4, 0, 0)
09/09 03:29:45 PM gpu device = cuda:0
09/09 03:29:45 PM config = {'drop_path_prob': 0.014487619036879763, 'grad_clip_value': 5, 'initial_lr': 1.4726143797983852e-05, 'n_conv_layers': 6, 'optimizer': 'adad', 'scheduler': 'Cosine', 'weight_decay': 0.0003157232243741241}
48 48 16
48 64 16
64 64 32
64 128 32
128 128 64
128 256 64
09/09 03:29:46 PM param size = 0.255930MB
09/09 03:29:46 PM WORKER: registered result for job (4, 0, 0) with dispatcher
09/09 03:29:46 PM job (4, 0, 0) failed with exception
Traceback (most recent call last):
  File "/home/rkohli/anaconda3/lib/python3.7/site-packages/hpbandster/core/worker.py", line 206, in start_computation
    result = {'result': self.compute(*args, config_id=id, **kwargs),
  File "darts-bohb.py", line 95, in compute
    if config['lr_schedular'] == 'Cosine':
KeyError: 'lr_schedular'

09/09 03:29:46 PM WORKER: start processing job (5, 0, 0)
09/09 03:29:46 PM gpu device = cuda:0
09/09 03:29:46 PM config = {'drop_path_prob': 0.10273677082671924, 'grad_clip_value': 4, 'initial_lr': 0.09444428220050997, 'n_conv_layers': 4, 'optimizer': 'adad', 'scheduler': 'Exponential', 'weight_decay': 0.0004333188616468378}
48 48 16
48 64 32
64 128 64
128 256 64
09/09 03:29:46 PM param size = 0.213546MB
09/09 03:29:46 PM WORKER: registered result for job (5, 0, 0) with dispatcher
09/09 03:29:46 PM job (5, 0, 0) failed with exception
Traceback (most recent call last):
  File "/home/rkohli/anaconda3/lib/python3.7/site-packages/hpbandster/core/worker.py", line 206, in start_computation
    result = {'result': self.compute(*args, config_id=id, **kwargs),
  File "darts-bohb.py", line 95, in compute
    if config['lr_schedular'] == 'Cosine':
KeyError: 'lr_schedular'

09/09 03:29:46 PM WORKER: start processing job (6, 0, 0)
09/09 03:29:46 PM gpu device = cuda:0
09/09 03:29:46 PM config = {'drop_path_prob': 0.11622380120970335, 'grad_clip_value': 6, 'initial_lr': 6.430354392936732e-05, 'n_conv_layers': 6, 'optimizer': 'adad', 'scheduler': 'Exponential', 'weight_decay': 0.00016655247746955444}
48 48 16
48 64 16
64 64 32
64 128 32
128 128 64
128 256 64
09/09 03:29:46 PM param size = 0.255930MB
09/09 03:29:46 PM WORKER: registered result for job (6, 0, 0) with dispatcher
09/09 03:29:46 PM job (6, 0, 0) failed with exception
Traceback (most recent call last):
  File "/home/rkohli/anaconda3/lib/python3.7/site-packages/hpbandster/core/worker.py", line 206, in start_computation
    result = {'result': self.compute(*args, config_id=id, **kwargs),
  File "darts-bohb.py", line 95, in compute
    if config['lr_schedular'] == 'Cosine':
KeyError: 'lr_schedular'

09/09 03:29:46 PM WORKER: start processing job (7, 0, 0)
09/09 03:29:46 PM gpu device = cuda:0
09/09 03:29:46 PM config = {'drop_path_prob': 0.28170165369686234, 'grad_clip_value': 4, 'initial_lr': 0.00012920551334604663, 'n_conv_layers': 5, 'optimizer': 'sgd', 'scheduler': 'Cosine', 'weight_decay': 5.1811281234748186e-05, 'nesterov': 'False', 'sgd_momentum': 0.4712382156071811}
48 48 16
48 64 32
64 128 32
128 128 64
128 256 64
09/09 03:29:47 PM param size = 0.245898MB
09/09 03:29:47 PM WORKER: registered result for job (7, 0, 0) with dispatcher
09/09 03:29:47 PM job (7, 0, 0) failed with exception
Traceback (most recent call last):
  File "/home/rkohli/anaconda3/lib/python3.7/site-packages/hpbandster/core/worker.py", line 206, in start_computation
    result = {'result': self.compute(*args, config_id=id, **kwargs),
  File "darts-bohb.py", line 95, in compute
    if config['lr_schedular'] == 'Cosine':
KeyError: 'lr_schedular'

09/09 03:29:47 PM WORKER: start processing job (8, 0, 0)
09/09 03:29:47 PM gpu device = cuda:0
09/09 03:29:47 PM config = {'drop_path_prob': 0.20157387451797604, 'grad_clip_value': 5, 'initial_lr': 0.0035939594455589316, 'n_conv_layers': 5, 'optimizer': 'adad', 'scheduler': 'Cosine', 'weight_decay': 2.007861759350898e-05}
48 48 16
48 64 32
64 128 32
128 128 64
128 256 64
09/09 03:29:47 PM param size = 0.245898MB
09/09 03:29:47 PM WORKER: registered result for job (8, 0, 0) with dispatcher
09/09 03:29:47 PM job (8, 0, 0) failed with exception
Traceback (most recent call last):
  File "/home/rkohli/anaconda3/lib/python3.7/site-packages/hpbandster/core/worker.py", line 206, in start_computation
    result = {'result': self.compute(*args, config_id=id, **kwargs),
  File "darts-bohb.py", line 95, in compute
    if config['lr_schedular'] == 'Cosine':
KeyError: 'lr_schedular'

09/09 03:29:47 PM WORKER: start processing job (9, 0, 0)
09/09 03:29:47 PM gpu device = cuda:0
09/09 03:29:47 PM config = {'drop_path_prob': 0.07193822049192287, 'grad_clip_value': 5, 'initial_lr': 0.018322912828280018, 'n_conv_layers': 4, 'optimizer': 'adam', 'scheduler': 'Plateau', 'weight_decay': 2.040848878245504e-05}
48 48 16
48 64 32
64 128 64
128 256 64
09/09 03:29:47 PM param size = 0.213546MB
09/09 03:29:47 PM WORKER: registered result for job (9, 0, 0) with dispatcher
09/09 03:29:47 PM job (9, 0, 0) failed with exception
Traceback (most recent call last):
  File "/home/rkohli/anaconda3/lib/python3.7/site-packages/hpbandster/core/worker.py", line 206, in start_computation
    result = {'result': self.compute(*args, config_id=id, **kwargs),
  File "darts-bohb.py", line 95, in compute
    if config['lr_schedular'] == 'Cosine':
KeyError: 'lr_schedular'

09/09 03:29:47 PM WORKER: start processing job (10, 0, 0)
09/09 03:29:47 PM gpu device = cuda:0
09/09 03:29:47 PM config = {'drop_path_prob': 0.025413500861081274, 'grad_clip_value': 6, 'initial_lr': 1.741668674571824e-05, 'n_conv_layers': 6, 'optimizer': 'sgd', 'scheduler': 'Cosine', 'weight_decay': 2.6299253429220064e-05, 'nesterov': 'False', 'sgd_momentum': 0.8585110190226121}
48 48 16
48 64 16
64 64 32
64 128 32
128 128 64
128 256 64
09/09 03:29:47 PM param size = 0.255930MB
09/09 03:29:47 PM WORKER: registered result for job (10, 0, 0) with dispatcher
09/09 03:29:48 PM job (10, 0, 0) failed with exception
Traceback (most recent call last):
  File "/home/rkohli/anaconda3/lib/python3.7/site-packages/hpbandster/core/worker.py", line 206, in start_computation
    result = {'result': self.compute(*args, config_id=id, **kwargs),
  File "darts-bohb.py", line 95, in compute
    if config['lr_schedular'] == 'Cosine':
KeyError: 'lr_schedular'

09/09 03:29:48 PM WORKER: start processing job (11, 0, 0)
09/09 03:29:48 PM gpu device = cuda:0
09/09 03:29:48 PM config = {'drop_path_prob': 0.26338065955443746, 'grad_clip_value': 5, 'initial_lr': 2.479212024127846e-05, 'n_conv_layers': 4, 'optimizer': 'sgd', 'scheduler': 'Plateau', 'weight_decay': 8.50174713585828e-05, 'nesterov': 'False', 'sgd_momentum': 0.26524061206685606}
48 48 16
48 64 32
64 128 64
128 256 64
09/09 03:29:48 PM param size = 0.213546MB
09/09 03:29:48 PM WORKER: registered result for job (11, 0, 0) with dispatcher
09/09 03:29:48 PM job (11, 0, 0) failed with exception
Traceback (most recent call last):
  File "/home/rkohli/anaconda3/lib/python3.7/site-packages/hpbandster/core/worker.py", line 206, in start_computation
    result = {'result': self.compute(*args, config_id=id, **kwargs),
  File "darts-bohb.py", line 95, in compute
    if config['lr_schedular'] == 'Cosine':
KeyError: 'lr_schedular'

09/09 03:29:48 PM WORKER: start processing job (12, 0, 0)
09/09 03:29:48 PM gpu device = cuda:0
09/09 03:29:48 PM config = {'drop_path_prob': 0.14412281091978443, 'grad_clip_value': 8, 'initial_lr': 0.006144502242189067, 'n_conv_layers': 4, 'optimizer': 'adam', 'scheduler': 'Cosine', 'weight_decay': 1.7552420079054188e-05}
48 48 16
48 64 32
64 128 64
128 256 64
09/09 03:29:48 PM param size = 0.213546MB
09/09 03:29:48 PM WORKER: registered result for job (12, 0, 0) with dispatcher
09/09 03:29:48 PM job (12, 0, 0) failed with exception
Traceback (most recent call last):
  File "/home/rkohli/anaconda3/lib/python3.7/site-packages/hpbandster/core/worker.py", line 206, in start_computation
    result = {'result': self.compute(*args, config_id=id, **kwargs),
  File "darts-bohb.py", line 95, in compute
    if config['lr_schedular'] == 'Cosine':
KeyError: 'lr_schedular'

09/09 03:29:48 PM WORKER: start processing job (13, 0, 0)
09/09 03:29:48 PM gpu device = cuda:0
09/09 03:29:48 PM config = {'drop_path_prob': 0.35861821865892923, 'grad_clip_value': 4, 'initial_lr': 0.0001518007963978253, 'n_conv_layers': 3, 'optimizer': 'sgd', 'scheduler': 'Exponential', 'weight_decay': 0.000789803314618894, 'nesterov': 'False', 'sgd_momentum': 0.6778674239283964}
48 48 16
48 64 32
64 128 64
09/09 03:29:48 PM param size = 0.118122MB
09/09 03:29:48 PM WORKER: registered result for job (13, 0, 0) with dispatcher
09/09 03:29:48 PM job (13, 0, 0) failed with exception
Traceback (most recent call last):
  File "/home/rkohli/anaconda3/lib/python3.7/site-packages/hpbandster/core/worker.py", line 206, in start_computation
    result = {'result': self.compute(*args, config_id=id, **kwargs),
  File "darts-bohb.py", line 95, in compute
    if config['lr_schedular'] == 'Cosine':
KeyError: 'lr_schedular'

09/09 03:29:48 PM WORKER: start processing job (14, 0, 0)
09/09 03:29:48 PM gpu device = cuda:0
09/09 03:29:48 PM config = {'drop_path_prob': 0.3105656338160384, 'grad_clip_value': 5, 'initial_lr': 0.0018915370128483732, 'n_conv_layers': 4, 'optimizer': 'adad', 'scheduler': 'Cosine', 'weight_decay': 0.0004803869326371755}
48 48 16
48 64 32
64 128 64
128 256 64
09/09 03:29:48 PM param size = 0.213546MB
09/09 03:29:48 PM WORKER: registered result for job (14, 0, 0) with dispatcher
09/09 03:29:49 PM job (14, 0, 0) failed with exception
Traceback (most recent call last):
  File "/home/rkohli/anaconda3/lib/python3.7/site-packages/hpbandster/core/worker.py", line 206, in start_computation
    result = {'result': self.compute(*args, config_id=id, **kwargs),
  File "darts-bohb.py", line 95, in compute
    if config['lr_schedular'] == 'Cosine':
KeyError: 'lr_schedular'

09/09 03:29:49 PM WORKER: start processing job (15, 0, 0)
09/09 03:29:49 PM gpu device = cuda:0
09/09 03:29:49 PM config = {'drop_path_prob': 0.3053866130200645, 'grad_clip_value': 5, 'initial_lr': 0.00025399020737262417, 'n_conv_layers': 5, 'optimizer': 'adam', 'scheduler': 'Cosine', 'weight_decay': 0.00013522565086677806}
48 48 16
48 64 32
64 128 32
128 128 64
128 256 64
09/09 03:29:49 PM param size = 0.245898MB
09/09 03:29:49 PM WORKER: registered result for job (15, 0, 0) with dispatcher
09/09 03:29:49 PM job (15, 0, 0) failed with exception
Traceback (most recent call last):
  File "/home/rkohli/anaconda3/lib/python3.7/site-packages/hpbandster/core/worker.py", line 206, in start_computation
    result = {'result': self.compute(*args, config_id=id, **kwargs),
  File "darts-bohb.py", line 95, in compute
    if config['lr_schedular'] == 'Cosine':
KeyError: 'lr_schedular'

09/09 03:29:49 PM WORKER: start processing job (16, 0, 0)
09/09 03:29:49 PM gpu device = cuda:0
09/09 03:29:49 PM config = {'drop_path_prob': 0.27333372917160315, 'grad_clip_value': 5, 'initial_lr': 3.7154305451153036e-05, 'n_conv_layers': 3, 'optimizer': 'adad', 'scheduler': 'Cosine', 'weight_decay': 0.0009195032692167355}
48 48 16
48 64 32
64 128 64
09/09 03:29:49 PM param size = 0.118122MB
09/09 03:29:49 PM WORKER: registered result for job (16, 0, 0) with dispatcher
09/09 03:29:49 PM job (16, 0, 0) failed with exception
Traceback (most recent call last):
  File "/home/rkohli/anaconda3/lib/python3.7/site-packages/hpbandster/core/worker.py", line 206, in start_computation
    result = {'result': self.compute(*args, config_id=id, **kwargs),
  File "darts-bohb.py", line 95, in compute
    if config['lr_schedular'] == 'Cosine':
KeyError: 'lr_schedular'

09/09 03:29:49 PM WORKER: start processing job (17, 0, 0)
09/09 03:29:49 PM gpu device = cuda:0
09/09 03:29:49 PM config = {'drop_path_prob': 0.3115288222872898, 'grad_clip_value': 8, 'initial_lr': 6.247686290881208e-05, 'n_conv_layers': 5, 'optimizer': 'adad', 'scheduler': 'Plateau', 'weight_decay': 0.0006831751448733878}
48 48 16
48 64 32
64 128 32
128 128 64
128 256 64
09/09 03:29:49 PM param size = 0.245898MB
09/09 03:29:49 PM WORKER: registered result for job (17, 0, 0) with dispatcher
09/09 03:29:49 PM job (17, 0, 0) failed with exception
Traceback (most recent call last):
  File "/home/rkohli/anaconda3/lib/python3.7/site-packages/hpbandster/core/worker.py", line 206, in start_computation
    result = {'result': self.compute(*args, config_id=id, **kwargs),
  File "darts-bohb.py", line 95, in compute
    if config['lr_schedular'] == 'Cosine':
KeyError: 'lr_schedular'

09/09 03:29:50 PM WORKER: start processing job (18, 0, 0)
09/09 03:29:50 PM gpu device = cuda:0
09/09 03:29:50 PM config = {'drop_path_prob': 0.07719896292954341, 'grad_clip_value': 5, 'initial_lr': 0.05623235154425975, 'n_conv_layers': 6, 'optimizer': 'sgd', 'scheduler': 'Cosine', 'weight_decay': 1.889258676096754e-05, 'nesterov': 'False', 'sgd_momentum': 0.8423885980368}
48 48 16
48 64 16
64 64 32
64 128 32
128 128 64
128 256 64
09/09 03:29:50 PM param size = 0.255930MB
09/09 03:29:50 PM WORKER: registered result for job (18, 0, 0) with dispatcher
09/09 03:29:50 PM job (18, 0, 0) failed with exception
Traceback (most recent call last):
  File "/home/rkohli/anaconda3/lib/python3.7/site-packages/hpbandster/core/worker.py", line 206, in start_computation
    result = {'result': self.compute(*args, config_id=id, **kwargs),
  File "darts-bohb.py", line 95, in compute
    if config['lr_schedular'] == 'Cosine':
KeyError: 'lr_schedular'

09/09 03:29:50 PM WORKER: start processing job (19, 0, 0)
09/09 03:29:50 PM gpu device = cuda:0
09/09 03:29:50 PM config = {'drop_path_prob': 0.08883677405354243, 'grad_clip_value': 5, 'initial_lr': 0.000818971999191841, 'n_conv_layers': 3, 'optimizer': 'adad', 'scheduler': 'Plateau', 'weight_decay': 1.5380795344975696e-05}
48 48 16
48 64 32
64 128 64
09/09 03:29:50 PM param size = 0.118122MB
09/09 03:29:50 PM WORKER: registered result for job (19, 0, 0) with dispatcher
09/09 03:29:50 PM job (19, 0, 0) failed with exception
Traceback (most recent call last):
  File "/home/rkohli/anaconda3/lib/python3.7/site-packages/hpbandster/core/worker.py", line 206, in start_computation
    result = {'result': self.compute(*args, config_id=id, **kwargs),
  File "darts-bohb.py", line 95, in compute
    if config['lr_schedular'] == 'Cosine':
KeyError: 'lr_schedular'

09/09 03:29:50 PM DISPATCHER: Dispatcher shutting down
09/09 03:29:50 PM DISPATCHER: shut down complete
